infrastructure/images/moduletest-reset-job/templates/moduletest-reset-job.template.yaml

```diff
@@ -6,18 +6,18 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: file-reset-{{ . }}
+  name: moduletest-reset-{{ . }}
   labels:
-    app: moduletest-file-reset-job
+    app: moduletest-reset-job
```

Comment:
What are you using this label for?

Reply:
Acutally not really anything. I should probably just remove it